components/membership/group-policy-impl/src/main/kotlin/net/corda/membership/impl/grouppolicy/GroupPolicyProviderImpl.kt

```
@@ -127,7 +128,12 @@ class GroupPolicyProviderImpl @Activate constructor(
 
         override fun getGroupPolicy(
             holdingIdentity: HoldingIdentity
-        ) = groupPolicies.computeIfAbsent(holdingIdentity) { parseGroupPolicy(it) }
+        ) = try {
+            groupPolicies.computeIfAbsent(holdingIdentity) { parseGroupPolicy(it) }
+        } catch (e: BadGroupPolicyException) {
```

Should we catch all the exceptions?

---

Good point. Added default catch-all handler.